### PR TITLE
Fix protobuf version

### PR DESCRIPTION
**Pull request by turian**
See https://github.com/Lightning-AI/lightning-transformers/pull/284#issuecomment-1243050284

This is based upon @rohitgr7 's awesome https://github.com/Lightning-AI/lightning-transformers/pull/284

### CODE REVIEW

**Comment by turian** [comment]
@rohitgr7 @carmocca @Borda Any idea why my CI is failing? Is it the skipped tests or the warnings? I didn't grep for any errors.

Even if @rohitgr7 's https://github.com/Lightning-AI/lightning-transformers/pull/284 gets merged, I still would like to understand how to placate CI in case I do future PRs.

**Comment by carmocca** [comment]
Looks like the tests pass, but there's an error either in the coverage computation or on pytest teardown.

I would start by removing coverage and see if the result is different: https://github.com/Lightning-AI/lightning-transformers/actions/runs/3033498716/workflow#L76-L77

**Comment by turian** [comment]
> Looks like the tests pass, but there's an error either in the coverage computation or on pytest teardown.
> 
> I would start by removing coverage and see if the result is different: https://github.com/Lightning-AI/lightning-transformers/actions/runs/3033498716/workflow#L76-L77

Thanks for the pointer. But where in the [CI logs](https://github.com/Lightning-AI/lightning-transformers/runs/8304991380?check_suite_focus=true) are you looking specifically, besides the final non-descript 'Error'?

**Comment by carmocca** [comment]
> besides the final non-descript 'Error'?

Just here, there's nothing else. This is why I suspect the command does not finish properly. As I pointed out it's coverage calling pytest. So even if pytest passes, coverage could fail (my hypothesis)

To check it, do

```diff
-coverage run --source lightning_transformers -m py.test lightning_transformers tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
+python -m py.test lightning_transformers tests -v
-coverage XML
```

**Comment by Borda** [comment]
@turian the needed proval is GH policy notning that we came up :)

File: .github/workflows/ci_testing.yml [hunk 1]
```diff
@@ -73,8 +73,10 @@ jobs:
 
     - name: Tests
       run: |
-        coverage run --source lightning_transformers -m py.test lightning_transformers tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
-        coverage xml
+        # suggested by https://github.com/Lightning-AI/lightning-transformers/pull/286#issuecomment-1243954206
```
Comment (carmocca) [inline .github/workflows/ci_testing.yml:76]: Looks like tests passed succesfully without `coverage` (obviously failed later when trying to upload it)

So the issue is probably caused by a bug in coverage.
We could try updating the test dependencies: https://github.com/Lightning-AI/lightning-transformers/blob/master/requirements/test.txt to match those in PL https://github.com/Lightning-AI/lightning/blob/master/requirements/pytorch/test.txt

**Comment by Borda** [comment]
I guess this is not needed any more, correct? :rabbit:

**Comment by turian** [comment]
> @turian, I guess this is not needed anymore after #284 was merged, correct? 🐰

Correct

**Pull request closed without merging**